test/reducers/user_vote_counter_test.js

```diff
@@ -0,0 +1,103 @@
+import userVoteCounter from "../../web/static/js/reducers/user_vote_counter"
+
+describe("userVoteCounter reducer", () => {
+  describe("when there is an empty action", () => {
+    describe("when no new state is passed", () => {
+      it("should return an empty object", () => {
+        expect(userVoteCounter(undefined, {})).to.deep.equal({})
+      })
+    })
+  })
+
+  describe("when there is a 'ENGAGE_LUNAR_LANDING' action", () => {
```

Comment:
i think this `describe` can be deleted, as it's definitely an unrecognized action type, which is covered in the `describe` below

Reply:
done!